### DISCUSSION TITLE
What is this error in ”OUTPUT”

### DISCUSSION THREAD

**Discussion by MilackB** [jilleb/MQB-FPA#65] [Q&A]
After runnning template, OUTPUT shows "*WARNING Line 223: Optimizing array of structures may cause incorrect results. Use <optimize=true|false> to override. "
What is this error?
![Error](https://user-images.githubusercontent.com/96638842/184465387-916ff4a0-66b1-4121-b36c-bae5779fb7af.PNG)

**Comment by jilleb** [comment]
That's no big deal, and more of a warning than an error, generated by 010 editor. It doesn't hurt your dataset.

**Reply by MilackB** [reply]
Hello jilleb! Thank you  for your reeplying. It's relief. Thank you so much.
